### PR TITLE
Adding information about latest Tag Manager changes

### PR DESCRIPTION
**Pull request by snake14**
### Description:

Updating the documentation with information about the latest changes to make `_paq.push()` work with MTM.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: docs/5.x/tagmanager/faq.md [hunk 1]
```diff
@@ -18,4 +18,8 @@ For example if your tag is implemented in `PopupTag.web.js`, you may provide a `
 
 If you self host your Matomo on premise, you can configure the list of available environments in the plugin settings by going to "Administration => Plugin Settings".
 
-The "live" environment will always be present.
\ No newline at end of file
+The "live" environment will always be present.
+
+## Why do I need to update the Tag Manager JavaScript when adding a new tracker configuration?
+
+The Matomo Configuration type Variables defined in a container and referenced by a Tag result in JavaScript trackers. When a tracker configuration is set using `_paq.push()`, that configuration is applied to all trackers created by Matomo tag manager and can override the values set in the Matomo Configuration Variables. If you create a new tracker configuration or find one that isn't working while using Matomo Tag Manager, please update `plugins/TagManager/Template/Tag/MatomoTag.web.js` so that the configuration is applied to all JavaScript trackers created by Tag Manager.
```
Comment (AltamashShaikh) [inline docs/5.x/tagmanager/faq.md]: should we mention the line also or something to search in the file like check implementation for `requireCookieConsentIndex` too learn how to make the change ?

Comment (AltamashShaikh) [reply docs/5.x/tagmanager/faq.md]: The Matomo Configuration type Variables -> The Matomo Configuration type variable defined 
Looks more apt ?

**Review by AltamashShaikh — COMMENTED** [review]
left 2 comments to check @snake14

File: docs/5.x/tagmanager/faq.md (hunk 1 above)
Comment (snake14) [reply docs/5.x/tagmanager/faq.md]: Thanks @AltamashShaikh. I didn't want to mention a line number as that could change. I guess I could mention a specific config to look for and use as an example.

Comment (snake14) [reply docs/5.x/tagmanager/faq.md]: I went ahead and updated the text to make it more consistent.

**Review by AltamashShaikh — APPROVED** [review]

**Comment by AltamashShaikh** [comment]
@snake14 We will also have 1 more release before 5.x-dev, and we are adding documentation only for 5.x-dev, I think good to add it in 4.x-dev and mention above version `4.13.3` we support this

**Review by AltamashShaikh — CHANGES_REQUESTED** [review]
Add same for 4.x-dev doc

**Comment by snake14** [comment]
> Add same for 4.x-dev doc

Thank you @AltamashShaikh. Please take a look and see if the most recent changes look alright.

**Review by AltamashShaikh — APPROVED** [review]

**Pull request merged**